src/site/_includes/components/component-preview/index.ts

```diff
@@ -1,3 +1,6 @@
+// @ts-expect-error iframe-resizer is missing types for v4
+import iframeResize from 'iframe-resizer/js/iframeResizer'
```

Comment:
Not a fan of this but I imported it like this because of module side effect behaviour running code for the content window.